spec/suites/map/MapSpec.js

```diff
@@ -791,6 +791,36 @@ describe("Map", function () {
 		});
 	});
 
+	describe('#padding options types', function () {
+		beforeEach(function () {
+			// flyToBounds needs a map container with non-null area
+			container.style.width = container.style.height = '100px';
+			map.setView([50.5, 30.51], 15);
+		});
+
+		var bounds = L.latLngBounds(L.latLng(1, 102), L.latLng(11, 122));
+		var paddingError = 'Padding must be a point type.';
+
+		['padding', 'paddingTopLeft', 'paddingBottomRight'].forEach(function (prop) {
+			var paddingOptions = {};
+			paddingOptions[prop] = 50; // invalid
```

Comment:
In fact there is a lot of other invalid values possible, like `[50]` or `"[50]"`. Not sure that we should check over all of them.
But if we not - then better refine the comment to something like "// most common misuse"...

Reply:
The intention is not to test for all possible invalid values, so I'm happy to update the comment.